python/selfplay/consecutive/README.md

```diff
@@ -0,0 +1,21 @@
+# Consecutive training using one machine
+
+You can train your own network using these scripts on one machine, by switching from selfplay to training instead of training in parallel.
+
+## Instructions
+
+Either copy the files to another folder or run them from where they are. Copy a relevant configuration file and rename it `keeper1.cfg` to this folder. Copy a relevant selfplay configuration file and rename it `selfplay1.cfg` to the same folder. Edit the paths in `nohuploop.sh` to reflect the correct location of your selfplay data and katago folder/executable.
+
+## Suggested settings for learning
+
+In `shuffle.sh` you should edit the min rows to 10000 because we're going to be working with less data to start. Expand window per row should be set to 1, taper window exponent to 1. In train.py you should decrease the samples per epoch to 250000 since we will generate fewer games to have a smaller learning loop. In your own copy of `selfplay1.cfg` you should set the amount of games to 3000, keeping cheap visits on. I suggest you reduce the visits in `keeper1.cfg` down to 100 or lower, because it cuts into your selfplay time significantly.
+
+## Modifications for smaller board sizes.
+
+In both config files you should change the settings. Set `bSizes` to the board size you wish to train, `bSizeRelProbs` should be set to `1`. Set `komiAllowIntegerProb` to `1.0` to allow for draws. `handicapProb` should be `0.0` on boards smaller than 19x19 because it's too imbalanced.
```

Comment:
komiAllowIntegerProb defaults to 1.0 now and is not even present in the config file by default in the first place now.
handicapProb should still make sense on 13x13 and larger, particularly when handicapCompensateKomiProb is large (so that komi is set to offset it and make the game fair). Also I would think handicapCompensateKomiProb should even not be set to exactly 1, just so the bot has some minimal continual amount of data about massively imbalanced board positions and a way of learning how to play in those settings without going crazy. Score utility means there's still learning signal in those positions, for both policy and for value-related outputs.